### PR TITLE
Attempt to fix pre-render errors

### PR DESCRIPTION
**Pull request by gomorizsolt**
I'm just guessing because the error itself is somewhat vague. Tried to follow their [recommendations](https://github.com/zeit/next.js/blob/master/errors/prerender-error.md) to find out what the root cause is but to no avail (it's an error that cannot be reproduced locally).

The issue definitely has something to do with: https://github.com/c-hive/dotdev/pull/131/files

My gut feeling is that there's something wrong with that "_app" component though it's identical to their [example](https://nextjs.org/docs/advanced-features/custom-app). Let's remove this and inject the [`<Head />`](https://nextjs.org/docs/api-reference/next/head) component in the root path (mostly because it's a single-page app without different routes with different titles).

### CODE REVIEW

**Comment by thisismydesign** [comment]
What are pre-render errors? Do we have an issue for this?

Did you test this on your fork? Does it solve the issue?

**Comment by gomorizsolt** [comment]
I should've been a bit more precise regarding where the issue stems from, sorry.

**TL;DR**

There's nothing to do with this PR. It should be fixed by #138.

> What are pre-render errors? Do we have an issue for this?

The [CI fails on master](https://github.com/c-hive/dotdev/runs/688348041?check_suite_focus=true#step:6:30) for some reason:

```
Error occurred prerendering page "/". Read more: https://err.sh/next.js/prerender-error
```

"Pre-render" is a term which is frequently used in the case of SSGs (it's one of their fundamental reason why they're so popular). What it basically refers to is generating each page *way ahead of time* (pages are compiled to static HTML) which results in 1) decreased loading time (because the app can be server from the "edge", be cached etc) 2) better user-experience (there's something to look at while the page is loading). One thing that's important to note here is that even though the pages are static, they're hydrated to become a fully interactive application if needed (e.g. fetch data at client-side when a component mounts).

Resources if you're interested in this topic:
- https://nextjs.org/docs/basic-features/pages#pre-rendering
- https://joshwcomeau.com/react/the-perils-of-rehydration/

> Did you test this on your fork? Does it solve the issue?

Right, it's my bad. I should've tested the changes against my forked repo instead of "guessing".

Turns out, this does *not* fix the error whatsoever:

```
TypeError: Cannot read property 'name' of null
```

https://github.com/c-hive/dotdev/blob/d05189401629373bfdf5671ba8285d402e54ea0d/pages/index.js#L11

[As we pointed out the other day](https://c-hive.slack.com/archives/CLGES7BAL/p1589197332014400), the `CONFIG` secret becomes a single-line string and because there's a leading `#` [in the first line](https://github.com/c-hive/dotdev/blob/50d899af20ab3472a280c71adf561ca84f754860/config/config.example.yml#L1) (because I copy-pasted the example config to the environment variable) it's converted to a single-line comment. That's the reason for receiving `null` when importing the config.

**Comment by gomorizsolt** [comment]
Closed because of https://github.com/c-hive/dotdev/pull/137#issuecomment-631300061.
